src/main/java/pages/Homepage.java

```diff
@@ -23,28 +26,45 @@ public class Homepage {
+    private final By pantsBannerImage = By.cssSelector(".block-promo.home-pants > img");
     private final By teesBanner = By.cssSelector(".block-promo.home-t-shirts");
     private final By teesBannerTopText = By.cssSelector(".block-promo.home-t-shirts .content .title");
     private final By teesBannerMainText = By.cssSelector(".block-promo.home-t-shirts .content .info");
     private final By teesBannerBottomText = By.cssSelector(".block-promo.home-t-shirts .content .action");
-    private final By teesBannerImageSource = By.cssSelector(".block-promo.home-t-shirts .image > img");
+    private final By teesBannerImage = By.cssSelector(".block-promo.home-t-shirts .image > img");
     private final By erinBanner = By.cssSelector(".block-promo.home-erin");
     private final By erinBannerTopText = By.cssSelector(".block-promo.home-erin .title");
     private final By erinBannerMainText = By.cssSelector(".block-promo.home-erin .info");
     private final By erinBannerBottomText = By.cssSelector(".block-promo.home-erin .action");
-    private final By erinBannerImageSource = By.cssSelector(".block-promo.home-erin > img");
+    private final By erinBannerImage = By.cssSelector(".block-promo.home-erin > img");
     private final By ecoFriendlyBanner = By.cssSelector(".block-promo.home-eco");
     private final By ecoFriendlyBannerTopText = By.cssSelector(".block-promo.home-eco .title");
     private final By ecoFriendlyBannerMainText = By.cssSelector(".block-promo.home-eco .info");
     private final By ecoFriendlyBannerBottomText = By.cssSelector(".block-promo.home-eco .action");
-    private final By ecoFriendlyBannerImageSource = By.cssSelector(".block-promo.home-eco >img");
+    private final By ecoFriendlyBannerImage = By.cssSelector(".block-promo.home-eco > img");
     private final By performanceBanner = By.cssSelector(".block-promo.home-performance");
     private final By performanceBannerTopText = By.cssSelector(".block-promo.home-performance .title");
     private final By performanceBannerMainText = By.cssSelector(".block-promo.home-performance .info");
     private final By performanceBannerBottomText = By.cssSelector(".block-promo.home-performance .action");
-    private final By performanceBannerImageSource = By.cssSelector(".block-promo.home-performance >img");
+    private final By performanceBannerImage = By.cssSelector(".block-promo.home-performance > img");
+    private final By hotSellersTitle = By.cssSelector(".content-heading .title");
+    private final By whatIsTrendingText = By.cssSelector(".content-heading .info");
+    private final By hotSellersProductsList = By.cssSelector(".product-items.widget-product-grid");
+    public static By hotSellersProductImage = By.className("product-image-photo");
+    public static By hotSellersProductDetails = By.className("product-item-details");
+    public static By hotSellersProductTitle = By.className("product-item-name");
```

Comment:
1. It should be public static final and names should be capitalized, example: HOT_SELLERS_PRODUCT_IMAGE.
Then getters are not needed, as it's static fields, so they can be used in test class.
element.findElement(Homepage.HOT_SELLERS_PRODUCT_IMAGE).
2. All public fields should be displayed above private fields in a class.